charts/seqr/values.yaml

```diff
@@ -84,9 +83,6 @@ affinity: |-
               {{- include "seqr.selectorLabels" . | nindent 12 }}
           topologyKey: "kubernetes.io/hostname"
 
-# -- Pod tolerations for the seqr Deployment
-tolerations: []
```

Comment:
We're not using tolerations... I think it's easier to support only `nodeSelector` and `affinity`?

Reply:
Same thought as the deploymentAnnotations here -- it's kind of hard to predict what tolerations an end user might need, so the general practice is provide an empty default that allows people to set arbitrary tolerations.

e.g. Azure kubernetes service might taint whatever they call spot nodes, and require deployments to tolerate that if you want to run the workload there... but we don't necessarily want to encode that in this chart, or default to tolerating that.